reference/tools.rst

```diff
@@ -1747,3 +1747,20 @@ for ``compiler.cppstd=17``, and so on.
 
 Parameters:
     - **settings** (Required): Conanfile settings. Use ``self.settings``.
+
+
+.. _tools.msvs_toolset:
+
+tools.msvs_toolset():
+---------------------
+
+.. code-block:: python
+
+    def msvs_toolset(settings)
+
+Returns the corresponding Visual Studio platform toolset based on the settings. For instance, it may return ``v142``
+for ``compiler=Visual Studio`` with ``compiler.version=16``. If ``compiler.toolset`` was set in settings, it has a
+priority and always returned.
```

Comment:
this means that the tool has no effect and it just returns the ``compiler.toolset`` value, right?

Reply:
correct, the only effect is returning `compiler.toolset` value in this case